Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageLayoutOptions.swift

```diff
@@ -4,120 +4,114 @@
 
 import Foundation
 
+public typealias ChatMessageLayoutOptions = Set<ChatMessageLayoutOption>
+
+public extension ChatMessageLayoutOptions {
+    var rawValue: String {
+        map(\.rawValue).joined(separator: "-")
```

Comment:
What is this useful for?

Reply:
This is used for the `reuseIdentifier` of the message cell. Depending on the options, it will have a different identifier

Reply:
I've added documentation ✅